res/qml/DeckInfoBar.qml

```diff
@@ -65,7 +65,9 @@ Rectangle {
         Mixxx.ControlProxy {
             group: root.group
             key: "play"
-            onValueChanged: spinnyIndicator.indicatorVisible = (value > 0)
+            onValueChanged: (value) => {
+                spinnyIndicator.indicatorVisible = (value > 0);
+            }
```

Comment:
With Qt6, `qmllint` warns about unqualified access in signal handlers:

```
Warning: ../res/qml/Sampler.qml:138:49: Unqualified access
                    playPositionControl.value = mouse.x / width;
Note: mouse is accessible in this scope because you are handling a signal at ../res/qml/Sampler.qml:136:32

Consider using a function instead
            onPositionChanged:  function(mouse) {...
```

Reply:
Thanks. So I guess this is also the style Qt then prefers. I'm happy with that decision.

Reply:
Oh, it even prints a warning on the terminal if you run the application:

     qt.qml.context:warning [Main] file:///home/jan/Projects/mixxx/res/qml/ControlKnob.qml:12:5 Parameter "value" is not declared. Injection of parameters into signal handlers is deprecated. Use JavaScript functions with formal parameters instead.